content/bn/docs/reference/glossary/cluster.md

```diff
@@ -4,20 +4,20 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   ওয়ার্কিং(working) মেশিনের একটি সেট, যাকে নোড বলা হয়, যা কন্টেইনারাইজড অ্যাপ্লিকেশন চালায়। প্রতিটি ক্লাস্টারে কমপক্ষে একটি ওয়ার্কার নোড থাকে।
+   ওয়ার্কার(worker) মেশিনের একটি সেট, যাকে নোড বলা হয়, যা কন্টেইনারাইজড অ্যাপ্লিকেশন চালায়। প্রতিটি ক্লাস্টারে কমপক্ষে একটি ওয়ার্কার নোড থাকে।
 
 aka: 
 tags:
 - fundamental
 - operation
 ---
-ওয়ার্কিং(working) মেশিনের একটি সেট, যাকে {{< glossary_tooltip text="নোডগুলো" term_id="node" >}} বলা হয়, 
+ওয়ার্কার(worker) মেশিনের একটি সেট, যাকে {{< glossary_tooltip text="নোড" term_id="node" >}} বলা হয়, 
 যা কন্টেইনারাইজড অ্যাপ্লিকেশন চালায়। প্রতিটি ক্লাস্টারে কমপক্ষে একটি ওয়ার্কার নোড থাকে।
 
 <!--more-->
-ওয়ার্কিং নোড(গুলো) হোস্ট করে {{< glossary_tooltip text="পডগুলো" term_id="pod" >}} যা হচ্ছে 
-অ্যাপ্লিকেশন এর কাজেরচাপ(workload) এর উপাদান। 
-{{< glossary_tooltip text="কন্ট্রোল প্লেন" term_id="control-plane" >}} পরিচালনা করে 
-ওয়ার্কার নোডগুলো(nodes) এবং পডগুলো(pods)। উৎপাদন পরিবেশে, কন্ট্রোল প্লেন(control plane) সাধারণত
-একাধিক কম্পিউটার জুড়ে চলে এবং একটি ক্লাস্টার সাধারণত একাধিক নোড চালায়, প্রদান করে
-ত্রুটি-সহনশীলতা(fault-tolerance) এবং উচ্চ প্রাপ্যতা(high-availability)।
+ওয়ার্কিং নোড(গুলো) {{< glossary_tooltip text="পড" term_id="pod" >}} হোস্ট করে যা
+অ্যাপ্লিকেশন ওয়ার্কলোড এর উপাদান ।
+{{< glossary_tooltip text="কন্ট্রোল প্লেন" term_id="control-plane" >}} ক্লাস্টারে থাকা
+ওয়ার্কার নোডগুলো এবং পডগুলো। প্রোডাকশন পরিবেশে, কন্ট্রোল প্লেন সাধারণত একাধিক
```

Comment:
ওয়ার্কার নোডগুলো এবং পডগুলো পরিচালনা করে  । প্রোডাকশন পরিবেশে, কন্ট্রোল প্লেন সাধারণত একাধিক

Reply:
sorry  it is my mistake